src/analytics/Properties.tsx

```diff
@@ -1323,10 +1323,13 @@ interface CICOEventsProperties {
   [CICOEvents.persona_kyc_error]: undefined
 }
 
-interface DappEventProperties {
+interface DappProperties {
   categoryId: string
   dappId: string
   dappName: string
+}
+
+interface DappEventProperties extends DappProperties {
   section: DappSection
```

Comment:
Why not keeping this even for favorite / unfavorite?

Reply:
good question, i didn't include these because the favourite / unfavourite actions can only come from one place (we're removing the dapp from the "all" list on favourite for MVP after all...) and the horizontalPosition was for recently used dapps only

Reply:
Ok, but aren't we gonna use a horizontal scrollview for favorites on the home screen?

Reply:
yes absolutely, but i haven't created events for that yet. those events can use the `DappEventProperties` type i think